app/domain/authentication/authn_jwt/input_validation/parse_mapped_claims.rb

```diff
@@ -0,0 +1,66 @@
+module Authentication
+  module AuthnJwt
+    module InputValidation
+      # Parse mapped-claims secret value and return a validated claims list
+      ParseMappedClaims ||= CommandClass.new(
+        dependencies: {
+          validate_claim_name: ValidateClaimName.new(
+            deny_claims_list_value: MANDATORY_CLAIMS_DENY_LIST
+          ),
+          logger: Rails.logger
+        },
+        inputs: %i[mapped_claims_secret_value]
+      ) do
+        def call
+          @logger.debug(LogMessages::Authentication::AuthnJwt::ParsingMappedClaims.new(@mapped_claims_secret_value))
+          validate_mapped_claims_secret_value_exists
+          validate_mapped_claims_list_values
+          @logger.debug(LogMessages::Authentication::AuthnJwt::ParsingMappedClaims.new(mapping_hash))
```

Comment:
Why same log message at beggining and end ? Shouldn't it be parsing and parsed successfuly?

Reply:
Done